newdle/client/src/components/creation/CreationSuccessPage.module.scss

```diff
@@ -10,9 +10,17 @@
 .newdle-link {
   display: flex;
   justify-content: center;
-  padding: 0.67857143em 1em;
-  border: 1px solid rgba(34, 36, 38, 0.15);
-  border-radius: 0.28571429rem;
+  width: 100%;
+  font-size: 1.5em !important;
+
+  input {
+    color: black !important;
+  }
+
+  button:active {
```

Comment:
not a big fan of this "animation" style... maybe using state to show a Popup is nicer after all? maybe we can actually go back to only having the copy button in this PR and then, if you want, you send another one that adds the popup?

Reply:
Let me try adding popup again and remove the animation.